### ISSUE TITLE
vmadm does not allow IPv6 resolvers

### ISSUE THREAD

**Issue by DvdGiessen** [joyent/smartos-live#531]
IPv6 resolvers are supported in [/etc/resolv.conf](https://smartos.org/man/4/resolv.conf), vmadm's validation marks these as invalid. Probably the validator needs to be tweaked to allow both IPv4 and IPv6 addresses for the `resolvers` property.

I couldn't immediately find where these validators are defined, but I'm sure someone with a bit more knowledge of the code will be able to fix this in mere minutes. :)

Stripped down examples:
IPv4 works:
```json
{
  "brand": "joyent",
  "image_uuid": "fec7e254-6e98-11e5-9284-3f9c607cb3e7",
  "resolvers": ["8.8.8.8"]
}
```

IPv6 doesn't work, but should:
```json
{
  "brand": "joyent",
  "image_uuid": "fec7e254-6e98-11e5-9284-3f9c607cb3e7",
  "resolvers": ["2001:4860:4860::8888"]
}
```
```
Invalid value(s) for: resolvers
```

**Comment by sjorge** [comment]
Just ran into this one too

**Comment by sjorge** [comment]
Here is the rfd for IPv6 support in SDC https://github.com/joyent/rfd/blob/master/rfd/0011/README.md, which also mentions the resolvers should support ipv6 resolvers.

**Comment by melloc** [comment]
@DvdGiessen and @sjorge , this is something that's planned, but I haven't done it yet. I've opened up a ticket to do it:

https://smartos.org/bugview/OS-5100

**Comment by sjorge** [comment]
@melloc: I noticed it was planned when I found the rfd (nice that they are public too!)

Left some other feedback on there too.

I had a poke around but the whole vmadm nodejs thing looks rather confusing to me. But's good to know it is on the drawing board.

**Comment by melloc** [comment]
Resolved in 921930a.
